src/toil/job.py

```diff
@@ -798,10 +848,12 @@ def effectiveRequirements(self, config):
         :rtype: Expando
         :return: a dictionary/object hybrid with one entry/attribute for each requirement
         """
+        if self._config is None:
```

Comment:
Is there any way we can eliminate this method? Maybe _config is already set in all call sites of this method?